docs/02-tutorials/01-intro/01-prci-set-up.mdx

```diff
@@ -33,42 +38,35 @@ As this build only integrates the _core_ parts of the library, external dependen
 Your specific context might be different. This is why each Privacy Component for the Web includes different builds and bundles covering any specific needs, as detailed in the [User Interfaces - Install](/docs/interfaces/install) section.
 
 :::
+-->
 
-Then, you only need to add the associated entry-point custom element `<bln-priv-request>` to your HTML document:
+## Add the Interface
 
-```html
-<h2 id="priv-request">Make a privacy request</h2>
+Now, let's expose the `<bldn-priv-request>` custom element:
 
-<bldn-priv-request></bldn-priv-request>
+```html
+<body>
+  <h1>Privacy Request Portal</h1>
+  <bldn-priv-request></bldn-priv-request>
+  <script src="https://cdn.jsdelivr.net/npm/@blindnet/prci@0.12.3/dist/index.all.min.js" type="module"></script>
+</body>
 ```
 
-The `bldn-priv-request` custom element support several attributes and properties to interact with it and configure its behavior.
+:::tip Checkpoint
+The HTML file with our changes thus far can be found [here](https://github.com/blindnet-io/devkit-quickstart/blob/main/web/expose-request-interface.html). You should now see a PRCI component which can be used to build requests. Once completed, the component will fire an [event](https://developer.mozilla.org/en-US/docs/Web/Events) containing the JSON representation of the request.
 
-Here, we only need to use the `computation-base-url` attribute to tell it to use our local PCE instance instead of the central one.
+See [**PRCI Events**](/docs/Interfaces/PRCI/Events) for more details on capturing this event.
+:::
 
-```html
-<bldn-priv-request computation-base-url="http://localhost:9000/v0"></bldn-priv-request>
-```
+Next, let's set up the [Privacy Computation Engine](./pce-set-up).
+
+<!--
+The `bldn-priv-request` custom element supports several attributes and properties to interact with it and configure its behavior.
 
-Leading us to the following HTML file:
+Here, we only need to use the `computation-base-url` attribute to tell it to use our local PCE instance instead of the central one.
```

Comment:
This is quickstart so we shouldn't require self-deploying the PCE. (Or is this part commented out?)

Reply:
I agree. I was thinking that you were writing the PCE part (where indeed we agreed not to require self-hosted PCE) but rather point them to an existing instance for testing, and move the part about running your own PCE instance to another section.

Reply:
This part is removed in the latest version of the quickstart (which is in develop now), but I guess the `improve-homepage` branch doesn't have those changes